docs/api/list.md

```diff
@@ -32,411 +25,36 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
-
-@Component({
-  tag: 'list-example',
-  styleUrl: 'list-example.css'
-})
-export class ListExample {
-  unread(ev: Event) {
-    console.log('Item is unread', ev);
-  }
-
-  render() {
-    return [
-      // List of Text Items
-      <ion-list>
-        <ion-item>
-          <ion-label>Pokémon Yellow</ion-label>
-        </ion-item>
-        <ion-item>
-          <ion-label>Mega Man X</ion-label>
-        </ion-item>
-        <ion-item>
-          <ion-label>The Legend of Zelda</ion-label>
-        </ion-item>
-        <ion-item>
-          <ion-label>Pac-Man</ion-label>
-        </ion-item>
-        <ion-item>
-          <ion-label>Super Mario World</ion-label>
-        </ion-item>
-      </ion-list>,
+Lists support several interactions including swiping items to reveal options, dragging to [reorder](./reorder) items within the list, and deleting items.
 
-      // List of Input Items
-      <ion-list>
-        <ion-item>
-          <ion-label>Input</ion-label>
-          <ion-input></ion-input>
-        </ion-item>
-        <ion-item>
-          <ion-label>Toggle</ion-label>
-          <ion-toggle slot="end"></ion-toggle>
-        </ion-item>
-        <ion-item>
-          <ion-label>Radio</ion-label>
-          <ion-radio slot="end"></ion-radio>
-        </ion-item>
-        <ion-item>
-          <ion-label>Checkbox</ion-label>
-          <ion-checkbox slot="start"></ion-checkbox>
-        </ion-item>
-      </ion-list>,
+## Basic Usage
 
-      // List of Sliding Items
-      <ion-list>
-        <ion-item-sliding>
-          <ion-item>
-            <ion-label>Item</ion-label>
-          </ion-item>
-          <ion-item-options side="end">
-            <ion-item-option onClick={(ev) => this.unread(ev)}>Unread</ion-item-option>
-          </ion-item-options>
-        </ion-item-sliding>
+import Basic from '@site/static/usage/list/basic/index.md';
 
-        <ion-item-sliding>
-          <ion-item>
-            <ion-label>Item</ion-label>
-          </ion-item>
-          <ion-item-options side="end">
-            <ion-item-option onClick={(ev) => this.unread(ev)}>Unread</ion-item-option>
-          </ion-item-options>
-        </ion-item-sliding>
-      </ion-list>
-    ];
-  }
-}
-```
+<Basic />
 
 
-</TabItem>
+## Inset List
 
+Adding the `inset` property to a list will apply margin around the list. In `ios` mode it will also add rounded corners to the list.
 
-<TabItem value="vue">
+import Inset from '@site/static/usage/list/inset/index.md';
 
-```html
-<template>
-  <!-- List of Text Items -->
-  <ion-list>
-    <ion-item>
-      <ion-label>Pokémon Yellow</ion-label>
-    </ion-item>
-    <ion-item>
-      <ion-label>Mega Man X</ion-label>
-    </ion-item>
-    <ion-item>
-      <ion-label>The Legend of Zelda</ion-label>
-    </ion-item>
-    <ion-item>
-      <ion-label>Pac-Man</ion-label>
-    </ion-item>
-    <ion-item>
-      <ion-label>Super Mario World</ion-label>
-    </ion-item>
-  </ion-list>
+<Inset />
```

Comment:
`md` mode shows an extra line below the first item here 🤔 Is this an Ionic bug?
![extra line](https://i.gyazo.com/67a432b488cdb16d0325ac3aab8b36e2.png)